devops/resolve_prereleases.rst

```diff
@@ -0,0 +1,62 @@
+.. _resolve_prereleases_summary:
+
+Handling version ranges and pre-releases
+========================================
+
+When developing a package and using version ranges for defining our dependencies,
+there might come a time when a new version of a dependency gets a new pre-release version that we would like to test
+before it's released to have a change to validate the new version ahead of time.
+
+At first glance, it could be expected that the new version matches our range if it intersect it,
+but :ref:`as described in the version ranges tutorial<tutorial_version_ranges_expressions>`,
+by default Conan does not match pre-release versions to ranges that don't specify it.
+Conan provides the :ref:`global.conf<reference_config_files_global_conf>` ``core.version_ranges:resolve_prereleases``,
+which when set to ``True``, enable pre-release matching in version ranges.
+This avoids having to modify and export the recipes of your dependency graph, which would become unfeasible for large ones.
+
+This conf has the added benefit of affecting the whole dependency graph, so that if any of our dependencies also define
+a requirement to our library of interest, the new version will also be picked up by it.
+
+Let's see this in action. Imagine we have the following (summarized) dependency graph,
+in which we depend on ``libpng`` and ``libmysqlclient``, both of which depend on ``zlib`` via the ``[>1.2 <2]`` version range:
+
+.. graphviz::
+
+    digraph G {
```

Comment:
It is possible to do a bit of styling, I think we have some in other parts of the docs if you want to copy it